README.md

```diff
@@ -24,15 +24,16 @@ Or install it yourself as:
 ## Configuration
 
 By default, feature_flagger uses the REDIS_URL env var to setup it's storage.
-You can configure this by using `configure` like such:
+You can configure this by using `configure`. Configure Redis directly on feature_flagger initializer, don’t reuse $redis like Sidekiq does.
```

Comment:
My suggestion:

>  You can set up FeatureFlagger by creating a file called `config/initializers/feature_flagger` with the following lines:
.....

I thinks this is more succint, what do you think?

Reply:
Agree! Removed the step 1 cause is an optional step. Check it!